Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "chiquito"
-version = "0.1.0"
+version = "0.1.2023082800"
```

Comment:
@qwang98 @trangnv do we need to update this for each release, or can the release come from the git tag?

Reply:
Cargo.toml triggers the pypi publication?

So to have a tag is a good practice, so we should have that. If it is still necessary to update this, perhaps we can have a script that creates the tag and also updates this with `sed` or from a template or something like that.

But would like to get explained how the CI pypi publication is triggered and create a markdown that explains how to create a release.